selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/dashboard/workspaces/Workspaces.java

```diff
@@ -200,6 +200,30 @@ public boolean isWorkspaceChecked(String workspaceName) {
     return Boolean.parseBoolean(attrValue);
   }
 
+  public void waitWorkspaceCheckboxEnabled(String workspaceName) {
+    webDriverWaitFactory
```

Comment:
Looks like we can simplify the code a bit if we will create and user method `SeleniumWebDriverHelper#waitCondition(ExpectedCondition<Boolean>)`

Reply:
I have created an issue for this task https://github.com/eclipse/che/issues/10105